static/styles/zulip.scss

```diff
@@ -1,5 +1,10 @@
 @import './reuseable_components.scss';
 
+/* Width of the message box left and right columns */
+$message_left_column_width: 46px;
+$message_right_column_width: 105px;
+$message_middle_column_width: calc(100% - #{$message_left_column_width} - #{$message_right_column_width});
```

Comment:
These values seem duplicated with the ones in `media.scss`; we should either make a new file with these values that both places can import, or add a comment mentioning these values should match the ones at the top of `media.scss`.

Reply:
Yeah looking into this. I think this gives us a good chance for cleanup. I was thinking that I'll add a comment for now and what we can do is create a `variables/` directory inside `static/styles/` and add files like `colors.scss` (since we were going to do the cleanup there as well) and `dimensions.scss` etc.?